lib/improver/utilities/save.py

```diff
@@ -151,7 +127,7 @@ def save_netcdf(cubelist, filename):
     for cube in cubelist:
         check_cube_not_float64(cube)
         _order_cell_methods(cube)
-        _check_for_units(cube)
+        enforce_units_and_dtypes(cube, enforce=False)
```

Comment:
Can we get rid of `check_cube_not_float64` with the new enforcement?

Reply:
I believe so. I did write as much in the comments on the ticket, I just suggested not doing it here. _check_for_units was only used here, so removing it was trivial. check_cube_not_float64 is used in several places and thus will be a messier change to remove. I didn't want to make this PR any larger.

Reply:
Ah yes I saw.  What I meant here was just can we remove the call from this function (at line 128).

Reply:
If you'd like. I was going to leave it to be all of a piece, but I shall remove it here.